control-plane/roles/monitoring/defaults/main/main.yaml

```diff
@@ -0,0 +1,7 @@
+---
+metal_monitoring_namespace: monitoring
+monitoring_ingress_dns: "monitoring.{{ metal_control_plane_ingress_dns }}"
+monitoring_tls: yes
+# monitoring_metal_metrics_exporter_version:
+monitoring_additional_ingress_annotations: {}
+metal_db_password: change-me
```

Comment:
Variables for a role should always be prefixed with the name of the role (style convention of this repository to counteract variable overlapping and re-use across multiple roles).

Reply:
https://github.com/metal-stack/metal-roles/commit/28e6db2dde255250f77c12f2341720cb3e3f8b88